tests/testthat/test_tidy.R

```diff
@@ -31,9 +31,22 @@ test_that("sample_n etc work", {
 })
 
 test_that("st_intersection of tbl returns tbl", {
- nc = read_sf(system.file("shape/nc.shp", package="sf")) 
+ nc = read_sf(system.file("shape/nc.shp", package="sf"))
  nc = st_transform(nc[1:3,], 3857)
  st_agr(nc) = "constant"
  expect_is(nc, "tbl_df")
  expect_is(st_intersection(nc[1:3], nc[4:6]), "tbl_df")
 })
+
+test_that("unnest works", {
```

Comment:
If you would make the test conditional on the tidyr version, it might pass tests and could be merged now.